### PR TITLE
Calibration benchmark

### PR DESCRIPTION
**Pull request by Cortys**
- Added a `base` method which trains a model without applying any method
- Label relaxation and label smoothing can now be used as alternative loss functions for all methods that use CE by default (to evaluate a "pure" label relaxation/smoothing approach, simply combine `method=base` with `supervised_loss=label_relaxation`/`supervised_loss=label_smoothing`)
- added a separate `calibrate.py` script which takes a pretrained model, wraps it with a post-hoc calibration method (currently only temp scaling supported), computes the calibration params based on the validation data and stores the calibration params as a separate artifact (downstream tasks can then load either the calibrated or uncalibrated model - can be set via `calibration=temperature_scaling`)
- CP is also in there, but "incorrectly" uses val for calibration right now, since we don't want to retrain all models just for CP. since CP won't be used for OOD and selective prediction, most likely this does not matter.

### CODE REVIEW

**Review by mmschlk — APPROVED** [review]
I think this is fine!
